model/src/main/java/org/eclipse/digitaltwin/aas4j/v3/model/OperationRequest.java

```diff
@@ -33,68 +33,51 @@ public interface OperationRequest {
      *
      * More information under https://admin-shell.io/aas/3/0/OperationRequest/inoutputArguments
      *
-     * @return Returns the OperationVariable for the property inoutputArguments.
+     * @return Returns the List of OperationVariables for the property inoutputArguments.
      */
     @IRI("https://admin-shell.io/aas/3/0/OperationRequest/inoutputArguments")
-    OperationVariable getInoutputArguments();
+    List<OperationVariable> getInoutputArguments();
 
     /**
      *
      * More information under https://admin-shell.io/aas/3/0/OperationRequest/inoutputArguments
      *
      * @param inoutputArguments desired value for the property inoutputArguments.
      */
-    void setInoutputArguments(OperationVariable inoutputArguments);
+    void setInoutputArguments(List<OperationVariable> inoutputArguments);
 
     /**
      *
      * More information under https://admin-shell.io/aas/3/0/OperationRequest/inputArguments
      *
-     * @return Returns the OperationVariable for the property inputArguments.
+     * @return Returns the List of OperationVariables for the property inputArguments.
      */
     @IRI("https://admin-shell.io/aas/3/0/OperationRequest/inputArguments")
-    OperationVariable getInputArguments();
+    List<OperationVariable> getInputArguments();
 
     /**
      *
      * More information under https://admin-shell.io/aas/3/0/OperationRequest/inputArguments
      *
      * @param inputArguments desired value for the property inputArguments.
      */
-    void setInputArguments(OperationVariable inputArguments);
+    void setInputArguments(List<OperationVariable> inputArguments);
 
     /**
      *
-     * More information under https://admin-shell.io/aas/3/0/OperationRequest/requestId
+     * More information under https://admin-shell.io/aas/3/0/OperationRequest/clientTimeoutDuration
      *
-     * @return Returns the String for the property requestId.
+     * @return Returns the String for the property clientTimeoutDuration.
      */
-    @IRI("https://admin-shell.io/aas/3/0/OperationRequest/requestId")
-    String getRequestId();
+    @IRI("https://admin-shell.io/aas/3/0/OperationRequest/clientTimeoutDuration")
+    String getClientTimeoutDuration();
```

Comment:
What about using java.time.Duration instead of string?
https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html

Reply:
@emildinchev please have a look again, I found a solution using javax.xml.datatype.Duration. I think it should work this way.

Reply:
Looks good to me!